src/uu/more/src/more.rs

```diff
@@ -536,6 +556,13 @@ impl<'a> Pager<'a> {
     }
 }
 
+fn paging_add_back_message(options: &Options, stdout: &mut std::io::Stdout) {
+    if options.lines.is_some() {
+        execute!(stdout, MoveUp(1)).unwrap();
+        stdout.write_all("\n\r...back 1 page\n".as_bytes()).unwrap();
```

Comment:
maybe handle the errors ?

Reply:
I use the `?` operator to propagate the `Err` result as a `UResult`.